### ISSUE TITLE
EventStoreExtension should not use Settings.Default

### ISSUE THREAD

**Issue by maciekciolek** [EventStore/EventStore.JVM#44]
Hello,
I met problems with using EventStoreExtension, because by default it configures ESConnection with Settings.Default:

```scala
class EventStoreExtension(system: ActorSystem) extends Extension {

  def settings: Settings = Settings.Default

  val actor: ActorRef = system.actorOf(ConnectionActor.props(settings), "eventstore-connection")

  val connection: EsConnection = new EsConnection(actor, system, settings.operationTimeout)
}
```
Setting.Default are built as follow:

```scala
lazy val Default: Settings = Settings(ConfigFactory.load())
```

It means that configuration will not origin from passed ActorSystem, but from application.conf. I suggest to create this Settings from system.settings.config.

**Comment by t3hnar** [comment]
@maciekciolek, yep I can change that EventStoreExtension use same conf as passed to ActorSystem, however I cannot change Settings.Default, as it is a static `Object`

**Comment by maciekciolek** [comment]
@t3hnar it would be great if EventStoreExtension will use config from ActorSystem. I think it is the best solution.

**Comment by maciekciolek** [comment]
@t3hnar can you provide me the time when are plan to fix it? Thank in advance for help.

**Comment by t3hnar** [comment]
@maciekciolek fixed in `v2.1.2`
